### PR TITLE
fileman: allow managing files in the app loader

### PR DESCRIPTION
**Pull request by devsnd**
This adds a small interface to the app loader to manage files in the app loader and not just on the watch. You can list, delete and show files in the loader and it shows the storage stats.

### CODE REVIEW

File: apps/fileman/ChangeLog [hunk 1]
```diff
@@ -1,3 +1,4 @@
 0.01: New app!
 0.02: Improve handling of large amounts of files (fix #579)
 0.03: Update RegExp use (Was using backreference instead of character code)
+0.04: Allow managing files in the app loader
```
Comment (nxdefiant) [inline apps/fileman/ChangeLog:4]: There is no need to increment the version number

File: apps/fileman/manage_files.html [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      getStats: function () {
+        Puck.eval('require("Storage").getStats()', (stats) => {
+          this.stats = Object.entries(stats);
+        });
+      },
+      deleteFile: function (file) {
+        if(confirm(`Really delete ${file}?`)) {
+          Puck.eval(`require("Storage").erase("${file}")`);
+          this.loadFiles();
+          this.getStats();
```
Comment (bobrippling) [inline apps/fileman/manage_files.html:65]: Do we want to do this after `Puck.eval`'s done? (in the callback)

**Review by bobrippling — COMMENTED** [review]
Looks good - do you have this deployed to your gh pages for testing? And as @nxdefiant said, we don't need the version bump

File: apps/fileman/ChangeLog (hunk 1 above)
Comment (devsnd) [reply apps/fileman/ChangeLog]: Ok, I removed it. It's true that there's nothing to update on the watch, but how are people going to know about this new feature in the app loader?

File: apps/fileman/manage_files.html (hunk 2 above)
Comment (devsnd) [reply apps/fileman/manage_files.html]: That's a bug! Thanks for the review, I fixed it.

File: apps/fileman/ChangeLog (hunk 1 above)
Comment (bobrippling) [reply apps/fileman/ChangeLog]: Same kind of way - they'd find new versions of the app in the app loader and they'll come across the interface changes there, although you are right in a way, in that they'd have to find the app in the app loader to start with.

Perhaps people would find it useful if the app loader summarised available `interface.html`s for their installed apps?

**Comment by bobrippling** [comment]
Thanks for the fix and the html!

**Pull request merged**